pkg/validate/validate_test.go

```diff
@@ -7,7 +7,6 @@ import (
 
 	"github.com/davecgh/go-spew/spew"
 	"github.com/ghodss/yaml"
-
```

Comment:
for better or worse as things stand we are using a convention of separating golang standard library imports, github.com/openshift/openshift-azure/* imports, and everything else into 3 separate blocks.